frictionless/portals/github/control.py

```diff
@@ -15,14 +15,38 @@ class GithubControl(Control):
     user: Optional[str] = None
     """NOTE: add docs"""
 
+    email: Optional[str] = None
+    """NOTE: add docs"""
+
     repo: Optional[str] = None
     """NOTE: add docs"""
 
+    formats: Optional[list] = [".csv"]
+    """NOTE: add docs"""
+
+    search: Optional[str] = None
+    """NOTE: add docs"""
+
+    per_page: Optional[int] = 30
+    """NOTE: add docs"""
+
+    page: Optional[int] = None
+    """NOTE: add docs"""
+
+    apikey: Optional[str] = None
```

Comment:
If you use ENV value here as a default you don't need to handle it anywhere else just use `control.apikey`

```
apikey: Optional[str] = os.environ.get("GITHUB_ACCESS_TOKEN")
```

Reply:
The same for email